spec/index.html

```diff
@@ -6814,12 +6818,16 @@ <h5>ENCODE_FOR_URI</h5>
           <section id="func-concat">
             <h5>CONCAT</h5>
             <pre class="prototype nohighlight"><span class="return">string literal</span>  <span class="operator">CONCAT</span>(<span class="type">string literal</span> <span>ltrl<sub>1</sub></span> ... <span class="type">string literal</span> <span>ltrl<sub>n</sub></span>)</pre>
-            <p>The <code>CONCAT</code> function corresponds to the XPath <a data-cite="XPATH-FUNCTIONS#func-concat">fn:concat</a> function. The function accepts string
-              literals as arguments.</p>
-            <p>The lexical form of the returned literal is obtained by concatenating the lexical
-              forms of its inputs. If all input literals are literals with identical language
-              tags, then the returned literal is a literal with the same language tag, in all
-              other cases, the returned literal is a literal with datatype <code>xsd:string</code>.</p>
+            <p>The <code>CONCAT</code> function returns a string literal following the 
+              <a href="#string-literal-return-type">string literal return type</a> rule.
+              The <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a> of the
+              returned string literal is obtained by concatenating the lexical
+              forms of the arguments of the function. 
+              If zero arguments are given, the result is an empty string literal without language tag.
+              If all input literals are literals with identical language tag,
+              then the returned literal is a literal with that language tag.
+              Otherwise, the returned literal is a literal with datatype <code>xsd:string</code>.
```

Comment:
```suggestion
              Otherwise, the returned literal is a literal with datatype <code>xsd:string</code>
              (and no language tag).
```